usagov/usagov-2021
# USAGOV-1479-Analytics-Reporter-Header

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1479

## Description


## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

- Easiest way to test this is using "Go Live" extension in vscode.
- From website-analytics/index.html use Go Live to open preview site.
- No data will be loaded that is fine for this ticket.
- The links "about this site" and "download data" should be on the same line as the dropdown
- The analytics dashboard H1 header should be under the blue line 


## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
